### DISCUSSION TITLE
Please help me

### DISCUSSION THREAD

**Discussion by kamal-ayman** [community/community#118871] [GitHub Education]
I am a student and my account is not accepted yet for a long time.. why ?

![Screenshot_2024-04-13-02-41-12-55_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/community/community/assets/52112682/f285fc1d-3d4d-48e0-bb21-fdc373e78618)

**Comment by eugene0928** [comment]
write to support

**Comment by samus-aran** [comment]
Hi @kamal-ayman 

It seems its still in a progressing stage but there isn't anything we can do on our end. Can you please navigate to the application page and confirm that you have submitted the form.

**Comment by fabyan09** [comment]
Helle @kamal-ayman,
I have the same Issue as you. If you gave a school Id, consider giving another proof or if you took a picture directly on the website, try to upload one next time, I'm sure this will help!
